### PR TITLE
BZ#1798848 Add step for setting mastersSchedulable param 4.3

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1798848

### CODE REVIEW

**Comment by codyhoag** [comment]
Preview: http://file.rdu.redhat.com/~choag/022120/mastersSchedulable-param-4.3/installing/installing_vsphere/installing-vsphere-network-customizations.html#modifying-nwoperator-config-startup_installing-vsphere-network-customizations

**Comment by codyhoag** [comment]
QE confirmation in [linked BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1798848).

File: modules/nw-modifying-operator-install-config.adoc [hunk 1]
```diff
@@ -46,6 +46,19 @@ $ ./openshift-install create manifests --dir=<installation_directory> <1>
...
++
+--
+.. Open the `manifests/cluster-scheduler-02-config.yml` file.
+.. Locate the `mastersSchedulable` parameter and set its value to `False`.
+.. Save and exit the file.
+--
++
+[NOTE]
+====
+Currently, due to a link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation], router Pods running on control plane machines will not be reachable by the ingress load balancer. This step might not be required in a future minor version of {product-title}.
```
Comment (bergerhoffer) [inline modules/nw-modifying-operator-install-config.adoc]: Recommend against saying `This step might not be required in a future minor version of {product-title}.` If it's no longer required in a future release, we should just make an edit to remove the step.

File: modules/nw-modifying-operator-install-config.adoc [hunk 2]
```diff
@@ -46,6 +46,19 @@ $ ./openshift-install create manifests --dir=<installation_directory> <1>
 <1> For `<installation_directory>`, specify the name of the directory that
 contains the `install-config.yaml` file for your cluster.
 
+. Modify the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines:
```
Comment (bergerhoffer) [inline modules/nw-modifying-operator-install-config.adoc]: Just wondering if you should specify where this file is? Is it in the `<installation_directory>/manifests/` directory as mentioned in step 3?

**Review by bergerhoffer — COMMENTED** [review]
Just a few comments, otherwise LGTM!

**Pull request merged**